src/Utilities/Helpers.js

```diff
@@ -15,6 +15,10 @@ import { Flex, FlexItem, Tooltip } from '@patternfly/react-core';
 import messages from '../Messages';
 import { intl } from './IntlProvider';
 
+export const isSet = (variable) => (
```

Comment:
Is there any reason why we need it?

Reply:
I also think that this is not needed. But the issue itself is weird https://sonarqube.corp.redhat.com/project/issues?id=insights%3Apatchman-ui&issues=AXfUchTERQz83K5m01H6&open=AXfUchTERQz83K5m01H6